config/initializers/okcomputer.rb

```diff
@@ -12,8 +12,8 @@
 OkComputer::Registry.register 'ruby_version', OkComputer::RubyVersionCheck.new
 # TODO: add app version check when okcomputer works with cap 3 (see http://github.com/sportngin/okcomputer#112)
 
-OkComputer::Registry.register 'purl_url', OkComputer::HttpCheck.new(Settings.purl_url)
-OkComputer::Registry.register 'stacks_url', OkComputer::HttpCheck.new(Settings.stacks_url)
+OkComputer::Registry.register 'purl_url', OkComputer::HttpCheck.new(Settings.purl_url) + 'status'
+OkComputer::Registry.register 'stacks_url', OkComputer::HttpCheck.new(Settings.stacks_url) + 'status'
```

Comment:
Do you mean to interpolate `'status` with the setting, or with the result of `OkComputer::HttpCheck.new`?